examples/1d/elixir_hypdiff_harmonic_nonperiodic.jl

```diff
@@ -0,0 +1,64 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the hyperbolic diffusion equations
+
+equations = HyperbolicDiffusionEquations1D(nu=1.25)
+
+initial_condition = initial_condition_harmonic_nonperiodic
+
+boundary_conditions = boundary_condition_harmonic_nonperiodic
+
+surface_flux = flux_lax_friedrichs
+solver = DGSEM(3, surface_flux)
+
+coordinates_min = -1
+coordinates_max =  2
```

Comment:
Is this test deliberately different from its 2D counterpart? In 2D, we have `nu = 1`, `polydeg = 4`, and a domain of `[0, 1]`. Just wondering what the reason for this non-symmetry is...

Reply:
Yeah, I made it deliberately different to have an example with an asymmetric domain. I also wanted to have an example where `nu` was not the default value to demonstrate this possibility (and increase coverage ;) ). I changed the `polydeg` to 3 because the time integrator from Hendrik is optimized for that value and I wanted this example to also demonstrate how one can use the homebrew time discretization. (This was also to teach myself how to call such functionality too)